frontend/src/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,165 @@
+import {useParams} from "react-router-dom";
+import {Avatar, Box, Container, Typography} from "@mui/material";
+import {User} from "../models/User";
+import {Loading} from "../components/Loading"
+import {useQuery} from "@tanstack/react-query";
+import axios from "axios";
+import {Colors} from "../assets/Colors";
+import "../App.css"
+import wave_mov from "../assets/wave_mov.svg"
+import wave_blue from "../assets/wave_blue.svg"
+import {DateTime} from "luxon";
+import {useState} from "react";
+
+
+function ProfilePage() {
+
+    const {userId} = useParams();
+    const [period, setPeriod] = useState("")
+    function time(unix: number) {
+        let date = DateTime.fromSeconds(unix);
+        let now = DateTime.now().setZone("system")
+        setPeriod(now.diff(date, ['minutes', 'seconds',"hours", "days"]).toFormat("dd : hh : mm : ss"))
+    }
+
+    const {isPending, error, data} = useQuery<User>({
+        queryKey: ['userData'],
+        queryFn: async () => {
+            let data: User = await axios.get('user/by-id/2')
+                .then(response => response.data)
+
+            data.icon = `http://127.0.0.1:5000/icon/by-id/${data.profile.icon_id}`;
+
+            time(data.profile.last_match_end)
+            setInterval(() => {time(data.profile.last_match_end)}, 10)
```

Comment:
Aici la delay-ul la `setInterval` cred ca e mai bine sa pui 1000 (ca sa fie 1 secunda). Altfel cred ca poate fi cam laggy daca se updateaza fiecare 10 milisecunde.

Adica sa fie asa:
```js
setInterval(() => {time(data.profile.last_match_end)}, 1000)
```

Reply:
Am facut si asa dar nu e in sincronizat cu ceasul real in cele mai multe cazuri. Face request dupa sau inainte de updateul minutului. De lag, nu mi s-a intamplat deloc, nu cred ca e un proces prea mare sa calculeze o ora.

Reply:
N-am înțeles exact la ce te referi. Vrei poate sa vb pe Discord mai in colo?

Reply:
sure

Reply:
Opțiunile cele posibile:
- faci o componenta separată pt timer/stopwatch/whatever
- crești delay-ul la 1000
- lași un comentariu gen "NOTE(sonia): dacă merge încet pagina, poate e de la asta"